app/models/financial_transaction.rb

```diff
@@ -4,6 +4,4 @@ class FinancialTransaction < ActiveRecord::Base
   belongs_to :transactable, polymorphic: true
 
   validates :rental_id, presence: true
-  validates :amount, presence: true
-  validates :adjustment, presence: true
```

Comment:
I understand why you eliminated this during test, but I think keeping this is fine!